app/frontend/package.json

```diff
@@ -82,10 +82,10 @@
     "@storybook/node-logger": "^6.1.2",
     "@storybook/preset-create-react-app": "^3.1.5",
     "@storybook/react": "^6.1.2",
-    "@testing-library/jest-dom": "^4.2.4",
-    "@testing-library/react": "^9.3.2",
-    "@testing-library/react-hooks": "^3.7.0",
-    "@testing-library/user-event": "^7.1.2",
+    "@testing-library/jest-dom": "^5.11.9",
```

Comment:
Took the opportunity to bump up a whole bunch of UI testing packages since I'm writing some and they are super outdated.